src/exo0 - Composing with pipe and flow/exo0.exercise.ts

```diff
@@ -42,9 +43,9 @@ export const not = (value: boolean) => !value;
 // `pipe` or `flow`), write the function `isOdd` that checks if a number is
 // odd.
 
-export const isOddP: (value: number) => boolean = unimplemented;
+export const isOddP = (value: number)  => pipe(value, isEven, not);
 
-export const isOddF: (value: number) => boolean = unimplemented;
+export const isOddF: (value: number) => boolean = value => flow(isEven, not)(value);
```

Comment:
This is correct, but to go even further you could write it like this:
```
export const isOddF: (value: number) => boolean = flow(isEven, not);
```
This is because `flow` will return a function -> so here, writing `flow(isEven, not)` will return a method expecting a `value: number` because this is what the first method passed in `flow` (so here `isEven`) is expecting!

Reply:
Thanks for the explanation! This is helpful :)